build/templates/mobsf-ios-scan.yml

```diff
@@ -0,0 +1,143 @@
+parameters:
```

Comment:
This file seems pretty similar to the android one. Could they be combined into one and parameterized instead of having 2?

Reply:
i'd suggest having both templates by platform just in case in the future we eant to add scan specific params for instance... make sense??

Reply:
Well it's more about whether the two files are almost identical right now and what they are about. If both files are the same they should be changed into a single file. If we want to add more scans in the future, I think the right way to do it would be to add more files for each type or security scan that we add. If the code is the same for MobSF for both iOS and Android, let's have a single file for MobSF. If we need different code for between iOS and Android for a Snyk scan, then we can have 2 files for Snyk. I don't think we should merge all scans into a file per platform (the code for MobSF is already pretty big on its own, I would not add another scan in that file).